src/Events/lib.mo

```diff
@@ -262,8 +262,8 @@ module {
     };
 
     public func inEventPeriod({ startsAt; endsAt } : MetaData, now : Time.Time) : Result.Result<(), Error> {
-        if (now < startsAt) return #err(#NotStarted(startsAt - now));
-        if (endsAt <= now)  return #err(#AlreadyOver(now - endsAt));
+        if (0 < startsAt and now < startsAt) return #err(#NotStarted(startsAt - now));
```

Comment:
This will work well, but I have used zero as a way to "close" events. Maybe we should just add a "delete event" method.

Reply:
I'll just use 1.